src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -68,10 +71,11 @@ export class UpperBarManager {
 
   private getControls(iconsOrder: IconsOrder): IconModel[] {
     const icons = Array.from(this.componentsRegistry.values());
-    return icons.sort((a, b) => (iconsOrder[a.label] > iconsOrder[b.label] ? 1 : -1));
+    return icons.sort((a, b) => (iconsOrder[a.displayName] > iconsOrder[b.displayName] ? 1 : -1));
```

Comment:
not critical, but you could also return iconsOrder[a.displayName] - iconsOrder[b.displayName]

Reply:
nice